ruby/audio/sdl.cpp

```diff
@@ -45,6 +47,11 @@ struct AudioSDL : AudioDriver {
     if(self.blocking) {
       while(SDL_GetQueuedAudioSize(_device) > _bufferSize) {
         //wait for audio to drain
+        auto bytesToWait = (_bufferSize / 4.0);
+        auto bytesPerSample = bitsPerSample / 8.0;
+        auto samplesRemaining = bytesToWait / bytesPerSample;
+        auto secondsRemaining = samplesRemaining / frequency;
+        usleep(secondsRemaining * 100000);
```

Comment:
Shouldn't this be `1000000`? To convert from seconds to microseconds.

Reply:
Yes, it should. How embarrassing! This probably means that I should only be sleeping for `1 / 40.0` of the buffer length instead of `1 / 4.0` to keep my testing results consistent. Thanks for the catch.